Source/x11/XGServerWindow.m

```diff
@@ -3235,7 +3235,7 @@ - (void) placewindow: (NSRect)rect : (int)win
   NSDebugLLog(@"XGTrace", @"DPSplacewindow: %@ : %d", NSStringFromRect(rect),
 	      win);
   nswin  = GSWindowWithNumber(win);
```

Comment:
Maybe we should move this line further down as nswin is no longer needed here and in many case it will never be needed at all.

Reply:
Sure, done.